airframe-control/src/test/scala/wvlet/airframe/control/ParallelTest.scala

```diff
@@ -97,6 +97,29 @@ class ParallelTest extends AirframeSpec {
       assert(List(Success(2), Failure(exception), Success(6)).forall(x => result.contains(x)))
     }
 
+    "be run for Seq using syntax sugar" in {
+      import wvlet.airframe.control.parallel._
+
+      val source = Seq(1, 2, 3)
+      val result: Seq[Int] = source.parallelMap(parallelism = 2){ x =>
```

Comment:
I think this syntax is a little bit unintuitive because the first argument is parallelism configuration, not the function to process the input elements (e.g., parallelMap(f: A=>B))

Reply:
OK, Let's consider more intuitive syntax following your comment below.